geomesa-tools/conf-filtered/geomesa-env.sh

```diff
@@ -143,7 +143,7 @@ function check_classpath() {
     group="${gav%%:*}"
     artifact="${gav#$group:}"
     artifact="${artifact%%:*}"
-    if [[ $classpath != *"$artifact"* ]]; then
+    if [[ ! $classpath =~ (^|:|/)${artifact}(-[^:]*)*\.jar ]]; then
```

Comment:
Ok, what corner case does this handle?

Reply:
basically stricter substring matching. iirc this was to fix commons-math/commons-math3 - previously `foo.jar:commons-math3.jar` would match for artifact `commons-math`, this makes it so it doesn't.

Reply:
and yes, that did cause me a 10 minute headache :P

Reply:
That only took 10 minutes to figure out?  I probably would have been stuck for much longer;).